.github/workflows/integration_test_secrets.yaml

```diff
@@ -0,0 +1,53 @@
+name: Integration tests with secrets
+
+on: workflow_call
+    
+jobs:
+  test-with-secrets:
```

Comment:
This was because of the issue of passing secrets in github workflows, since it will not get interpolated in arguments section.
If we were to pass this down and support it in operator-workflows integration tests, it would need to be specifically catered just for wordpress-k8s, would look something like below:
```
uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
with:
    secrets:
        TEST_ASKIMET_KEY: ${{ secrets.TEST_ASKIMET_KEY }}
        ... other secrets
```
This is why I've decided to have a separate marker for integration tests that require secrets(namely Askimet plugin and openID plugin) that helps leverage the existing integration_test workflow in operator-workflows and prevents the tests from diverging too much.
Would that help understand the situation? Open for any other suggestions as well! :)
Thank you for the comment!

Reply:
Understood

Reply:
I've been thinking about this and iirc you can pass the secrets just by adding `secrets: inherit` to the caller workflow. The secrets do not need to be explicitly declared.

Reply:
Yup, it works for the secrets getting used by the workflow (in this case `integeration_test.yaml` in operator-workflows), but it wouldn't get parsed correctly as argument due to how github passes secrets between workflows (good security though!).